README.md

```diff
@@ -1,123 +1,90 @@
-These instances, like their counterparts in static, have start and endtimes so
-that we don't send emails when the survey has closed.  Unlike their counterparts
-in static they do not have match rules on the path - response that gets past an
-`survey_id` check and is in the `active?` time period will be sent an email.
-
-The email is sent using GOV.UK Notify using the "GOV.UK Surveys" service and a
-hardcoded email template (name: `email_survey_signup`, id: `8fe8ab4f-a6ac-44a1-9d8b-f611a493231b`)
-that belongs to that service.  This means that all running instances must use
-API keys for the same service or the template won't exist.  The API key is
-provided with the env var:
-
-    SURVEY_NOTIFY_SERVICE_API_KEY
-
-Deployed environments have this filled in via puppet and our standard mechanism
-for handling keys.  On the GOV.UK dev vm you'll want to join the service on
-GOV.UK Notify and create your own  API key.  When creating a key for yourself
-choosese either:
-
-* "test" - which won't send any emails at all, but will give you valid
-           responses from the API
-* "team" - which will only send emails to people on the team or the whitelisted
-           email addresses.
-
-Note that future versions may allow for different surveys to use different
-templates, but they'll still all have to belong to the same Notify service.
-
-#### A note on Notify template parameters
-
-Notify templates can be parameterised, and when we talk to the notify API we
-send a `personalisation` key that contains values for all the parameters in the
-template.  Notify will error if there are missing keys, but it will also error
-if there are extra keys.  This means we have to take care when editing the
-template in the Notify UI and take care not to introduce, nor remove parameters
-without updating the code.
-
-Currently the template takes a single parameter:
-
-* `survey_url` - the url that the survey lives at and will be sent in the email
-                 to invite the user to take part in that survey - this is
-                 constructed by taking the `url` of the `EmailSurvey` instance
-                 and adding the `survey_source` as a `c` param to the query
-                 string. At the end of the url we will also add the `ga_client_id`
-                 (e.g. if we have ga_client_id = '12345.67899' then the resulting 
-                 `survey_url` will be appended with `&gcl=12345.67890`)
-
-Adding new parameters will require a deploy, so it might be best to add a new
-template with new parameters and have the deploy change the template id *and*
-the parameters.
+them a link to a survey. The survey will then be emailed to them via the 
+`feedback` app. Please be aware that the answers we get back on the survey will 
+NOT be stored in `feedback`.
+
+A more thorough explanation of how email surveys work can be found in the docs:
+[docs/email_survey_signup.md](docs/email_survey_signups.md).
+
+
+### A note on Notify template parameters
```

Comment:
This is only relevant to the `Email Survey Signups` section as that's what uses notify.  We can move this whole section into the the `docs/email_survey_signup.md` file.